src/Helpers/Changelog.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace PlacetoPay\AppVersion\Helpers;
+
+/**
+ * The Changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
+ */
+class Changelog
+{
+    public static function path(): string
+    {
+        return base_path('CHANGELOG.md');
+    }
+
+    public static function exists(): bool
+    {
+        return file_exists(self::path());
+    }
+
+    public static function read(): string
+    {
+        $content = 'Not available right now';
```

Comment:
Not Available

Reply:
Se cambio el mensaje y se creo como constante de la clase